packages/react-core/src/components/DescriptionList/__tests__/DescriptionListHelpTextButton.test.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { DescriptionListTermHelpTextButton } from '../DescriptionListTermHelpTextButton';
+
+import styles from '@patternfly/react-styles/css/components/DescriptionList/description-list';
+
+test('Renders to match snapshot', () => {
+  const { asFragment } = render(<DescriptionListTermHelpTextButton>test</DescriptionListTermHelpTextButton>);
+  expect(asFragment()).toMatchSnapshot();
+});
+
+test(`Renders default class ${styles.descriptionListText}`, () => {
+  render(<DescriptionListTermHelpTextButton>test</DescriptionListTermHelpTextButton>);
+  expect(screen.getByText('test')).toHaveClass(styles.descriptionListText);
```

Comment:
```suggestion
  expect(screen.getByText('test')).toHaveClass(styles.descriptionListText, {exact: true});
```

Also a test that props are spread correctly. File name should also be updated to `DescriptionListTermHelpTextButton`